src/snake/mrd_utils/loader.py

```diff
@@ -402,7 +403,48 @@ def get_kspace_frame(
             )
         return traj, data
 
+class SliceDataloader(MRDLoader):
+    """Load slice MRD files k-space frames iteratively."""
+
```

Comment:
Add a bit more documentation about this, mention that it augments a base data_loader for instance, and what it modifies compares to it.